metadata-service/graphql-servlet-impl/src/main/java/com/datahub/graphql/GraphQLController.java

```diff
@@ -137,10 +137,12 @@ CompletableFuture<ResponseEntity<String>> postGraphQL(HttpEntity<String> httpEnt
                     "Errors while executing graphQL query: %s, result: %s, errors: %s",
                     queryJson, executionResult.toSpecification(), executionResult.getErrors()));
           } else {
-            log.debug(
-                String.format(
-                    "Executed graphQL query: %s, result: %s",
-                    queryJson, executionResult.toSpecification()));
+            if (log.isDebugEnabled()) {
```

Comment:
Using a parameterized message would probably be more efficient here:

```
log.debug("Executed graphQL query: {}, result: {}", queryJson, () -> executionResult.toSpecification());
```

https://logging.apache.org/log4j/2.3.x/performance.html

Reply:
@bda618 I'm improving the GraphQL controller logging at #10404